shared/types/user.ts

```diff
@@ -13,7 +13,6 @@ export const UserBase = z.object({
   agency: AgencyBase.shape._id,
   betaFlags: z
     .object({
-      sgid: z.boolean().optional(),
     })
```

Comment:
is there a difference between `z.object({})` and `z.object()`?

Reply:
I receive an error if i did use `z.object()` instead. 
<img width="953" alt="image" src="https://user-images.githubusercontent.com/59867455/205859885-2a4d6c62-4d9e-4c0f-9e5d-45e46c544201.png">

The other option is to remove betaflags entirely, however, I don't think that will be a good option? As we will likely have future betas.

Reply:
sgtm, let's leave it as empty object